packages/wpcom-checkout/package.json

```diff
@@ -40,6 +40,7 @@
 		"@automattic/calypso-stripe": "^1.0.0",
 		"@automattic/composite-checkout": "^1.0.0",
 		"@automattic/shopping-cart": "^1.0.0",
+		"@emotion/core": "^10.0.27",
```

Comment:
This will need to be changed when https://github.com/Automattic/wp-calypso/pull/55736 merges